site/standalone/installing.md

```diff
@@ -25,6 +25,17 @@ $EDITOR ./deploy/flux-deployment.yaml
 kubectl apply -f ./deploy
 ```
 
+## Helm users
+
+Create all resources defined in the
+[deploy-helm directory](../../deploy-helm/).
+
+```
+$EDITOR ./deploy-helm/flux-deployment.yaml
+$EDITOR ./deploy-helm/helm-operator-deployment.yaml
+kubectl apply -f ./deploy-helm
+```
+
```

Comment:
The following now looks like it is in the "Helm users" section

Reply:
made clear what belongs where